packages/next/test/fixtures/36-instrumentation-hook/pages/api/hello.js

```diff
@@ -0,0 +1,7 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
Any reason why the docs are linked here?

Reply:
![copy-paste-paste-gif](https://user-images.githubusercontent.com/11064311/221611118-133483dd-81b8-417f-8640-70cea2d0f9ea.gif)